### DISCUSSION TITLE
ssh GUI file transfer client

### DISCUSSION THREAD

**Discussion by robbinslewis** [gravitational/teleport#8823] [Q&A]
Hello, We're investigating the use of teleport and one sticking point for our Windows user base is many are use to a GUI client. We can probably convince them to use Windows Terminal rather than PuTTY for cli sessions, but the same users rely on WinSCP, Filezilla, cyberduck, etc. for drag and drop file transfers and scp may be a tough sell.

Is anyone aware of GUI file transfer clients that will work with Teleport and/or ssh certificates in general?

Thank you.

**Comment by jwieder** [comment]
Visual Studio Code has a remote SSH extension that I can confirm works with my own Teleport environment. Generic instructions available here: https://goteleport.com/docs/server-access/guides/vscode/#prerequisites

**Comment by webvictim** [comment]
PuTTY has been updated to support OpenSSH certificates in version 0.78 now. Hopefully WinSCP will follow suit soon and then it'll be possible to use it with Teleport!

Update 15th May 2023:
Support for **user** certificates has been added to WinSCP: https://winscp.net/tracker/1873

Unfortunately, WinSCP will not work with Teleport until support for **host** certificates is also added: https://winscp.net/tracker/2145

**Reply by abolast** [reply]
this helped me a lot

**Comment by zmb3** [comment]
Our Teleport Connect GUI client also supports file transfers!

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/7527103/206747562-cbf6f445-94ea-4d15-ad99-f12aa045cb55.png">
